scripts/spack_packages/raja/package.py

```diff
@@ -282,6 +282,10 @@ def hostconfig(self, spec, prefix, py_site_pkgs_dir=None):
                 cuda_reldebinf_flags = "-O3 -g -Xcompiler -Ofast -Xcompiler -finline-functions"
                 cuda_debug_flags = "-O0 -g -Xcompiler -O0 -Xcompiler -finline-functions"
    
+            if ("clang" in cpp_compiler):
```

Comment:
Why turn on benchmarks in the spack package? In general, users won't want this. I think it would be better to pass CMake the option '-DENABLE_BENCHMARKS=On' for particular CI builds. Thoughts?

Reply:
I think this is already handled, if we set `tests=basic` in the spec: https://github.com/LLNL/RAJA/blob/cc7c874145772732e9efe24927ce0e1ea049910c/scripts/spack_packages/raja/package.py#L362

This will enable both normal tests and benchmarks.

Reply:
Good point, David. I changed it so it just uses the `tests` arg in the spec